<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QPushButton" name="pushButtonE">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Emergency STOP</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButton2">
      <property name="text">
       <string>Set power_SP</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>2. Power Set-Point</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>1. LV Voltage</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton1">
      <property name="text">
       <string>Read LV_U</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPlainTextEdit" name="text2"/>
    </item>
    <item row="1" column="0">
     <widget class="QTextBrowser" name="browser1"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>671</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="1">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>LV_I:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QTextBrowser" name="browserSP"/>
    </item>
    <item row="1" column="1">
     <widget class="QTextBrowser" name="browserI"/>
    </item>
    <item row="1" column="0">
     <widget class="QTextBrowser" name="browserU"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>LV_U:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>read SP:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QTextBrowser" name="browser4"/>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>answer:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>301</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>3. Control</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPlainTextEdit" name="text3U"/>
    </item>
    <item row="2" column="1">
     <widget class="QPlainTextEdit" name="text3P"/>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton3">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>U</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>P</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextBrowser" name="browserEmergency">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>671</width>
     <height>87</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>4. Any request</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTextEdit" name="text42"/>
    </item>
    <item row="2" column="0">
     <widget class="QTextEdit" name="text41"/>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton4">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>address aka transID:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>data:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>361</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="1">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Timer</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTextBrowser" name="timer3"/>
    </item>
    <item row="1" column="1">
     <widget class="QTextBrowser" name="timer2"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>T2:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>T3:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
